named-tunnel-k8s/cloudflared.yaml

```diff
@@ -84,6 +87,12 @@ data:
     # from the internet to cloudflared, run `cloudflared route dns <tunnel> <hostname>`.
     # E.g. `cloudflared route dns example-tunnel tunnel.example.com`.
     ingress:
+    # The first rule proxies traffic to the httpbin sample Service defined in app.yaml
     - hostname: tunnel.example.com
       service: http://web-service:80
+    # This rule sends traffic to the built-in hello-world HTTP server. This lets you check if cloudflared
+    # is up, without needing to use kubectl (e.g. if you're on mobile)
```

Comment:
Nothing against "pimping" this with the hello world service.  But why wouldn't the user be able to reach `tunnel.example.com` on his mobile as well?

Reply:
If cloudflared -> origin service is disrupted. I'll explain in the comment.